tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -22,7 +22,24 @@ public String name() {
   public void processFeature(SourceFeature sourceFeature, FeatureCollector features) {
     if (sourceFeature.canBeLine() && sourceFeature.hasTag("highway") &&
       !(sourceFeature.hasTag("highway", "proposed", "abandoned", "razed", "demolished", "removed", "construction"))) {
+
       String highway = sourceFeature.getString("highway");
+      String shield_text = sourceFeature.getString("ref");
+      String network_val = sourceFeature.getString("network");
+      shield_text = (shield_text == null ? null : shield_text.split(";")[0]);
+      if( shield_text != null ) {
+        if( shield_text.contains("US ") ) {
+          shield_text = shield_text.replaceAll("US ", "");
+          network_val = "US:US";
```

Comment:
<img width="430" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/66d51888-111e-489f-96d9-84af5833da85">

Reply:
This is the Tilezen way of specifying the network values, which is mostly true to OSM's values.